src/plonemeeting/portal/core/browser/manage_users.py

```
@@ -284,3 +337,289 @@ def handleInvite(self, action):
+        self.widgets["username"].value = username
+        self.widgets["username"].readonly = "readonly"
+        self.widgets["email"].value = member.getProperty("email", "")
+        self.widgets["email"].readonly = "readonly"
+        self.widgets["fullname"].value = member.getProperty("fullname", "")
+        self.widgets["fullname"].readonly = "readonly"
+        self.widgets["user_groups"].value = self.get_manageable_groups_for_user(username)
+
+    @button.buttonAndHandler(_plone("save"), name="save")
+    def handleSave(self, action):
+        """Update the SSO user's group membership (Keycloak owns the rest)."""
+        data, errors = self.extractData()
```

Ruff (0.16.1)

[warning] Unpacked variable `errors` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)